### PR TITLE
Added some preloading and prefetching for speed

### PR DESCRIPTION
**Pull request by rishFilet**
By fixing those loading issues it seemed to have sped up he loading of the website. I wrote about it in the wiki under `optimizations` but essentially is prefetching the fonts and checking to see if the components are loaded first before rendering the html, using the hook, effect.

Started also editing style a bit to match the style guide

### CODE REVIEW

**Comment by BBoutet1** [comment]
@rishFilet loading looks enhanced. But I don't understand why using "ClientSideRender" component both in "AboutUs/index.js" and in "layout/Mainlayout.js". In "Mainlayout.js" is already wrapping the entire page while in "AboutUs/index.js" is wrapping only the body.  Isn't it enough using ClientSideRender component only in "Mainlayout.js"? I approve the PR as you can make the small change the next PR if needed.

**Review by BBoutet1 — APPROVED** [review]
If needed correct the small detail I mentioned in comment on your next PR.

**Review by demiwu96 — COMMENTED** [review]
The Josefin Sans font doesn't show, except the first section of the page.

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
         "@material-ui/icons": "^4.9.1",
         "@mdx-js/mdx": "^1.6.16",
         "gatsby": "^2.24.54",
+        "gatsby-plugin-prefetch-google-fonts": "^1.4.3",
         "gatsby-plugin-web-font-loader": "^1.0.4",
```
Comment (harshvchawla) [inline package.json:22]: Should this go away then?

File: src/components/Navbar/Logo.js [hunk 2]
```diff
@@ -23,6 +25,7 @@ const useStyles = makeStyles({
     fontFamilly: "Poppins"
   },
 });
+
```
Comment (harshvchawla) [inline src/components/Navbar/Logo.js:28]: This file can be reverted as it only has whitespace changes?

**Review by harshvchawla — APPROVED** [review]
I am approving so that the PR doesn't depend/block on me.
I will need to study about hooks and pre-rendering which might take time
Have provided other generic comments

File: package.json (hunk 1 above)
Comment (rishFilet) [reply package.json:22]: You mean web font loader?

Comment (harshvchawla) [reply package.json:22]: Yeah
I didn't fetch/compile but saw the file diff and wondering if the prefetch-google-fonts is a substitute for web-font-loader

Comment (rishFilet) [reply package.json:22]: Yes it is, ill remove it in the next pr i am making for these changes

**Pull request merged**